rcode-reader/src/main/java/eu/rssw/pct/elements/EventElement.java

```diff
@@ -0,0 +1,75 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.DataType;
+import eu.rssw.pct.IParameter;
+import eu.rssw.pct.RCodeInfo;
+
+public class EventElement extends AbstractAccessibleElement {
+  // private int flags;
+  private int returnType;
+  private IParameter[] parameters;
+
+  private String returnTypename;
+  private String delegateName;
+
+  public EventElement(String name, Set<AccessType> accessType, byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
+    super(name, accessType);
+    
+    // this.flags = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(order).getShort();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)